### PR TITLE
Update DESCRIPTION

### PR DESCRIPTION
**Pull request by pawelru**
- be in line with whats in suggests
- cran type of reference is redundant

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -91,8 +91,8 @@ Config/Needs/verdepcheck: haleyjeppson/ggmosaic, tidyverse/ggplot2,
     tidymodels/broom, daattali/colourpicker, daattali/ggExtra,
     aphalo/ggpmisc, aphalo/ggpp, slowkow/ggrepel, baddstats/goftest,
     gridExtra, ramnathv/htmlwidgets, jeroen/jsonlite, yihui/knitr,
```
Comment (pawelru) [inline DESCRIPTION:93]:
```suggestion
   ramnathv/htmlwidgets, jeroen/jsonlite, yihui/knitr,
```

File: DESCRIPTION [hunk 2]
```diff
@@ -90,9 +90,9 @@ Config/Needs/verdepcheck: haleyjeppson/ggmosaic, tidyverse/ggplot2,
     insightsengineering/tern, tidyverse/tibble, tidyverse/tidyr,
     tidymodels/broom, daattali/colourpicker, daattali/ggExtra,
     aphalo/ggpmisc, aphalo/ggpp, slowkow/ggrepel, baddstats/goftest,
-    gridExtra, ramnathv/htmlwidgets, jeroen/jsonlite, yihui/knitr,
-    daroczig/logger, deepayan/lattice, MASS,
-    insightsengineering/nestcolor, r-lib/rlang, rstudio/rmarkdown,
+   ramnathv/htmlwidgets, jeroen/jsonlite, yihui/knitr,
```
Comment (pawelru) [inline DESCRIPTION:93]:
```suggestion
    ramnathv/htmlwidgets, jeroen/jsonlite, yihui/knitr,
```

File: DESCRIPTION [hunk 3]
```diff
@@ -90,9 +90,9 @@ Config/Needs/verdepcheck: haleyjeppson/ggmosaic, tidyverse/ggplot2,
     insightsengineering/tern, tidyverse/tibble, tidyverse/tidyr,
     tidymodels/broom, daattali/colourpicker, daattali/ggExtra,
     aphalo/ggpmisc, aphalo/ggpp, slowkow/ggrepel, baddstats/goftest,
-    gridExtra, ramnathv/htmlwidgets, jeroen/jsonlite, yihui/knitr,
-    daroczig/logger, deepayan/lattice, MASS,
-    insightsengineering/nestcolor, r-lib/rlang, rstudio/rmarkdown,
+    ramnathv/htmlwidgets, jeroen/jsonlite, yihui/knitr,
+    daroczig/logger, deepayan/lattice,
+    insightsengineering/nestcolor, r-lib/pkgload, r-lib/rlang, rstudio/rmarkdown,
     insightsengineering/rtables, tidyverse/rvest, sparkline,
```
Comment (pawelru) [inline DESCRIPTION:96]:
```suggestion
    insightsengineering/rtables, tidyverse/rvest, htmlwidgets/sparkline,
```

**Comment by donyunardi** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by donyunardi — APPROVED** [review]

**Pull request merged**